src/Theme/HookPreprocess.php

```diff
@@ -55,4 +58,59 @@ public static function block(array &$variables): void
             ->load($variables['elements']['#id']);
         $variables['block'] = new WrappedConfigEntity($block);
     }
+
+    /**
+     * @param array{
+     *   node: NodeInterface<string, FieldItemListInterface>,
+     *   teaser: bool,
+     *   url: string,
+     *   author_name?: MarkupInterface,
+     *   author_picture?: mixed[],
+     *   date?: MarkupInterface,
+     *   display_submitted?: bool,
+     *   label?: mixed[]
+     * } $variables
+     */
+    public static function node(array &$variables): void
+    {
+        $node = $variables['node'];
+        $variables['promote'] = $node->isPromoted();
+        $variables['sticky'] = $node->isSticky();
+        $variables['status'] = $node->isPublished();
+        $variables['preview'] = $node->in_preview ?? null;
+        $variables['name'] = $variables['author_name'] ?? '';
+        $variables['node_url'] = $variables['url'];
+        $variables['title'] = $variables['label'] ?? '';
+        $variables['submitted'] = '';
+        $variables['user_picture'] = '';
```

Comment:
What if someone's code already populated this? I know I'd customize it in my code. We're now ruining their customization.